Makefile

```diff
@@ -113,6 +116,7 @@ docker-test-integration:
 		-w /usr/src/github.com/vmware-tanzu/antrea \
 		-v $(DOCKER_CACHE)/gopath:/tmp/gopath \
 		-v $(DOCKER_CACHE)/gocache:/tmp/gocache \
+		-v $(CURDIR)/.coverage:/usr/src/github.com/vmware-tanzu/antrea/.coverage \
 		-v $(CURDIR):/usr/src/github.com/vmware-tanzu/antrea:ro \
```

Comment:
does that work?

Reply:
From the [log](https://github.com/vmware-tanzu/antrea/pull/1139/checks?check_run_id=1020358917), it works.

Reply:
Thanks for confirming. I had concerns about the order, and thought that the read only bind mount would hide the other one. But then I did some research and it seems volumes are mounted in lexicographic order (https://github.com/moby/moby/issues/26051#issuecomment-242847347) - first `/usr/src/github.com/vmware-tanzu/antrea` then `/usr/src/github.com/vmware-tanzu/antrea/.coverage` - so things make more sense to me now.

Reply:
Still not sure why codecov bot does not comment, I'm going to add a comment by Github Actions.